bundles/org.openhab.binding.bluetooth/src/main/java/org/openhab/binding/bluetooth/ConnectedBluetoothHandler.java

```diff
@@ -203,12 +204,14 @@ public void onCharacteristicReadComplete(BluetoothCharacteristic characteristic,
     @Override
     public void onCharacteristicWriteComplete(BluetoothCharacteristic characteristic,
             BluetoothCompletionStatus status) {
-        logger.debug("Wrote {} to characteristic {} of device {}: {}", characteristic.getByteValue(),
-                characteristic.getUuid(), address, status);
+        logger.debug("Wrote {} to characteristic {} of device {}: {}",
+                HexUtils.bytesToHex(characteristic.getByteValue()), characteristic.getUuid(), address, status);
     }
 
     @Override
     public void onCharacteristicUpdate(BluetoothCharacteristic characteristic) {
+        logger.debug("Received new value '{}' for characteristic '{}' of device '{}'",
+                HexUtils.bytesToHex(characteristic.getByteValue()), characteristic.getUuid(), address);
```

Comment:
I added logging to notifications since it read and write are logged but it seemed missing here.